Dnd/dndJavascript.js

```diff
@@ -10,44 +19,31 @@ function showSeason(item) {
     return season_number.replace(/\s+/g, '')
   }
   
-  function getSeasonContent() {
-    const season_number = getSelectedSeasonNumber();
+  function getSeasonContent(season_number) {
     const path = "EpisodeData/" + season_number + ".json"
     return fetch(path).then(function(response) {
       return response.json();
     });
   }
   
-  function clearEpisodes() {
-    const row = document.getElementById('row-of-cards');
-    if (row !== null) row.remove();  
-  }
-  
-  async function displaySeasonContent(){
-    const row_div = document.createElement("div");
-    row_div.className="row";
-    row_div.id = "row-of-cards" 
-    row_div.style.cssText="color: black;"
-  
-    const data = await getSeasonContent();
-    data.forEach(res => {
+  function displaySeasonContent(episodeData){
+    const container = document.getElementById("row-of-cards");
+    episodeData.forEach(episode => {
       const column_div = document.createElement("div");
       column_div.className="col-xl-3 col-lg-4 col-md-6";
+      column_div.id="dnd-column";
```

Comment:
I think this should probably be a class?

Reply:
I was reluctant to make it a class as there is no corresponding CSS for it. Is it normal to use class names purely as identifiers?